internal/util/paramtable/component_param.go

```diff
@@ -1396,6 +1398,16 @@ func (p *dataNodeConfig) initFlushInsertBufferSize() {
 	p.FlushInsertBufferSize = bs
 }
 
```

Comment:
the config point of delete buffer size, the default confined value here is 64MB per channel
The value may be too big for production environment